File a/bin/carla.lv2/resources is not a regular file or directory and was skipped
diff -ruN a/source/modules/water/files/File.cpp b/source/modules/water/files/File.cpp
--- a/source/modules/water/files/File.cpp	2024-09-21 23:56:30.000000000 +0200
+++ b/source/modules/water/files/File.cpp	2024-10-21 11:56:39.436559683 +0200
@@ -1222,13 +1222,8 @@
 //=====================================================================================================================
 namespace
 {
-   #ifdef CARLA_OS_LINUX
-    typedef struct stat64 water_statStruct;
-    #define WATER_STAT    stat64
-   #else
     typedef struct stat   water_statStruct;
     #define WATER_STAT    stat
-   #endif
 
     bool water_stat (const String& fileName, water_statStruct& info)
     {
